data/sacd.attestation.example.json

```diff
@@ -4,34 +4,31 @@
     "type": "dimo.sacd",
     "data": {
         "grantor": {
-            "address": "0x1110000000000000000000000000000000000000",
-            "name": "Optional Human Readable Name"
+            "address": "0x07B584f6a7125491C991ca2a45ab9e641B1CeE1b",
+            "name": "Alice"
         },
         "grantee": {
-            "address": "0x2220000000000000000000000000000000000000",
-            "name": "Optional Human Readable Name"
+            "address": "0x07B584f6a7125491C991ca2a45ab9e641B1CeE1b",
+            "name": "Bob"
         },
         "effectiveAt": "2025-03-11T14:30:00Z",
         "expiresAt": "2030-12-20T05:20:45Z",
-        "asset": "did:nft:137:0x3330000000000000000000000000000000000000_31415",
+        "asset": "did:erc721:80002:0x45fbCD3ef7361d156e8b16F5538AE36DEdf61Da8:928",
         "additionalDates": {},
         "agreements": [
             {
-                "type": "cloudevent", 
+                "type": "cloudevent",
                 "eventType": "dimo.attestation",
-                "source": "0x3330000000000000000000000000000000000000",
-                "ids": ["unique-attestation-id-1", "unique-attestation-id-2"],
-                "effectiveAt": "2025-03-11T14:30:00Z",
-                "expiresAt": "2030-12-20T05:20:45Z",
-                "signatures": [
-                    {
-                        "signer": "0x1110000000000000000000000000000000000000",
-                        "signature": "0x...",
-                        "timestamp": "2025-03-07T12:30:00Z"
-                    }
-                ]
+                "source": "0x07B584f6a7125491C991ca2a45ab9e641B1CeE1b",
+                "ids": [
+                    "unique-attestation-id-1",
+                    "unique-attestation-id-2"
+                ],
+                "effectiveAt": "2022-03-11T14:30:00Z",
+                "expiresAt": "2030-12-20T05:20:45Z"
             }
         ],
         "extensions": {}
-    }
+    },
+    "signature": "0x92e576fbce2c2c29ede118c8e674d4aae6f1e606f5f84c3096fbe962840f3ec608a961d90c40c3331de13ba29ae4526498fe2eed5dfd4486c3f3e36fd97715631c"
```

Comment:
Shouldn't it be "signatures" and an array? I understand this case is just one, but for consistency with the other examples.

Reply:
I thought we'd agreed we wanted to follow the structure of cloudevents (ie having a single "signature" field) but I do hear the point about consistency